bigtable/bigtable.go

```diff
@@ -278,13 +278,27 @@ func (t *Table) ReadRows(ctx context.Context, arg RowSet, f func(Row) bool, opts
 		return err
 	}, retryOptions...)
 
+	// Convert error to grpc status error
```

Comment:
also nit - is this logic elsewhere in the codebase? Might be nice to make it a util if so

Reply:
Not being used anywhere else